### PR TITLE
Conversion of Gdoc to Asciidoctor

### PR DESCRIPTION
**Pull request by thijs1108**
Used run.sh to convert all Gdoc files to Asciidoctor files. To view the result use `mvn` in `wicket-user-guide`

### CODE REVIEW

File: wicket-user-guide/pom.xml [hunk 1]
```diff
@@ -1,195 +1,357 @@
...
-						</executions>
-					</plugin>
-                </plugins>
-			</build>
-		</profile>
-	</profiles>
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.asciidoctor.maven</groupId>
+    <artifactId>asciidoc-to-html-example</artifactId>
```
Comment (martin-g) [inline wicket-user-guide/pom.xml]: Wrong groupId and artifactId

**Comment by klopfdreh** [comment]
The first commit is rather big so it is hard to find out - so my question: Is there any documentation about how to setup the php environment to convert gdoc to adoc, or is it simply copy the php files into the root folder of the php server? Thanks for the great work so far! :smiley:

**Comment by thijs1108** [comment]
The php scripts can be thrown away after the merge of this commits because then the documentation should be updated in the .adoc files and not in the .gdoc files anymore. 

To start converting all the gdoc files to adoc you need to pass all the gdoc files through the `adoccommandline.php`. `run.sh` is doing that, it's searching all the gdoc files in guide/ and convert it to adoc format in asciidoc/. So the first step is run `$ ./run.sh`. 

The next step is that there should be one index.adoc file which include all the other adoc files. Gdoc has toc.yml but asciidoc need a completely other syntax. `toctoindex.php` loads guide/toc.yml and give an output that should be index.adoc. To do that, run `$ ./toctoindex.php > asciidoc/index.adoc`.

If that's done, run `$ mvn` in `wicket-user-guide` to render index.adoc to index.html and index.pdf with all adoc files included. In `wicket-user-guide/target/generated-docs` is the result. Ignore the empty directories.

I hope this will help. Thanks for your response!

**Comment by martin-g** [comment]
Do we still have interest in this PR?
Anyone wants to merge it (properly) ?

**Comment by dashorst** [comment]
I still am interested in this. Will take a looksy when I have time.

**Comment by klopfdreh** [comment]
I think this PR can be closed, now. Thanks a lot for the great new guide!

**Comment by martin-g** [comment]
@thijs1108 @dashorst Please close this PR! It has been merged to 8.x and 7.x. Thanks!

**Pull request closed without merging**